static/docs/commands-reference/get.md

```diff
@@ -1,7 +1,7 @@
 # get
 
-Download or copy file or directory from another DVC repository (on a Git server
-e.g. Github) into the local file system.
+Download or copy file or directory from an external <abbr>DVC project</abbr>
+hosted on a Git repository online (e.g. Github) into the local file system.
```

Comment:
@shcheklein to your comment in https://github.com/iterative/dvc.org/commit/1fd6c115210ae422b71c9eba52b94e4d761ced91#r35008637,

> This not fully correct... I believe we can use `dvc get` on "offline" projects, not online only.
Most likely we can use `dvc get` with `.` (internal project) - a sample use case is to get an artifact from a specific revision to then diff it.

Indeed. I just confirmed all this. Added info explaining that `url` can be a local file system path and removed the term "external".

> `hosted on Git` sounds not common, better: `from Git`, `specified by Git URL`, etc.

Improved all, see my latest changes to the file (: 

BUT, do we want `dvc get` to work with `.`? Seems confusing and unnecessary to me. That's what `dvc checkout` is for... If so maybe I should add a note that you can even use this same current repo as a source and an example on how to replace checkout with get.

Reply:
> BUT, do we want dvc get to work with .? Seems confusing and unnecessary to me. That's what dvc checkout is for... If so maybe I should add a note that you can even use this same current repo as a source and an example on how to replace checkout with get. 

The difference is that you can get two (or more) versions of the same file/directory simultaneously with `dvc get. It can be useful to dig deeper, compare with some diff tool, reuse for whatever reason. On the other hand, with `dvc checkout` you can _checkout_ only one version at a time, also it's quite cumbersome and requires good knowledge of Git.

Reply:
> you can get two (or more) versions of the same file/directory simultaneously with `dvc get`.

Good point... (Using the `--out` option.) OK added an advance example about this to `dvc get` for PR #585. See https://github.com/iterative/dvc.org/pull/585/commits/8e9e94725675e5285d4d36f0a4fd8ae77c45e95b